tools/ci_build/install/ci_requirements.txt

```diff
@@ -2,3 +2,4 @@ pylint == 1.9 ; python_version <= '2.7'
 pylint == 2.3.1 ; python_version > '2.7'
 docformatter == 1.1
 yapf == 0.26.0
+testipynb
```

Comment:
Could you pin it to `0.0.2`. Just makes maintenance easier.

Reply:
done that